src/components/AmplitudeAnalytics/utils.ts

```diff
@@ -1,10 +1,14 @@
 import { Currency, CurrencyAmount, Percent, Token } from '@uniswap/sdk-core'
 
-export const getDurationTillTimestampSinceEpoch = (futureTimestampSinceEpoch?: number): number | undefined => {
+export const getDurationTillTimestampSinceEpochSeconds = (futureTimestampSinceEpoch?: number): number | undefined => {
```

Comment:
`getDurationSinceEpochSeconds`

Reply:
that's a little misleading, it's not duration since epoch, but duration TILL timestamp (in the future) denoted in seconds since epoch. 

my bad for not making the terminology clear.
changed it to getDurationTillTimestampSeconds, and input variable to futureTimestampInSecondsSinceEpoch for clarity.